catalog/task/create-pyxis-image/0.4/create-pyxis-image.yaml

```diff
@@ -44,7 +44,7 @@ spec:
   steps:
     - name: create-pyxis-image
       image:
-        quay.io/hacbs-release/release-utils@sha256:5298e31e7e7a97cab005750096abe8dbfca3f422cf049fd87de76a117072a9b5
+        "TODO: Update when the updated release-utils image is available"
```

Comment:
Edit this once https://github.com/redhat-appstudio/release-service-utils/pull/58 is merged.